src/pages/Signin/SignIn.tsx

```diff
@@ -0,0 +1,135 @@
+/** @jsxImportSource @emotion/react */
+import { css } from "@emotion/react";
+import colors from 'styles/color';
+
+import { useForm } from 'react-hook-form';
+import { zodResolver } from "@hookform/resolvers/zod";
+
+import { type SubmitHandler } from 'react-hook-form';
+
+import * as z from "zod";
+import { loginSchema } from 'schema';
+
+type SignInFormType = z.infer<typeof loginSchema>;
+
+const SignIn = () => {
+  const {
+    handleSubmit,
+    register,
+    reset,
+    formState: { errors },
+  } = useForm<SignInFormType>({
+    resolver: zodResolver(loginSchema),
+  });
+
+  const onSubmit: SubmitHandler<SignInFormType> = (data) => {
+    console.log(data);
+    reset();
+  }
+
+  return (
+    <form css={[_container]} onSubmit={handleSubmit(onSubmit)}>
+      <div css={_formContainer}>
+        <h2 css={_title}>Sign In</h2>
+        <input
+          css={_input}
+          type="text"
+          placeholder="Email"
+          {...register("accountId")}
```

Comment:
placeholder와 동일하게 email 정도로 표현해주면 어떨까요? email, password 짝이 저는 더 익숙한 것 같아요!
```suggestion
          placeholder="Email"
          {...register("email")}
```